lib/default-layout-config.js

```diff
@@ -8,25 +8,34 @@ module.exports = [
       },
       views: [
         {
-          name: "stdout"
+          type: "log",
+          title: "stdout",
+          borderColor: "green",
+          streams: ["stdout"]
         },
         {
-          name: "stderr"
+          type: "log",
+          title: "stderr",
+          borderColor: "red",
+          streams: ["stderr"]
         }
       ]
     },
     {
       views: [
         {
-          name: "cpu",
+          type: "cpu",
+          title: "cpu utilization",
           limit: 30
         },
         {
-          name: "eventLoop",
+          type: "eventLoop",
+          title: "EL delay",
```

Comment:
It's just the title doesn't fit in one line when the view is narrow, looks a bit untidy.

Oh, I know: how about we move counters to the title, so by default title would be `"eventloop delay ({delay}), high ({high})"`, and here we can hide the counters, leaving just `"eventloop delay"`?